contrib/sarplus/DEVELOPMENT.md

```diff
@@ -39,3 +39,33 @@ sbt test
 ```
 
 (use ~test and it will automatically check for changes in source files, but not build.sbt)
+
+
+## Notes for Spark 3.x  ##
+
+The code now has been modified to support Spark 3.x, and has been
+tested under different versions of Databricks Runtime (including 6.4
+Extended Support, 7.3 LTS, 9.1 LTS, 10.0 and 10.1) on Azure Databricks
+Service.  But now manual packaging is needed:
+
+
+```bash
+export VERSION=0.5.0
```

Comment:
Shouldn't VERSION be specified inside the code? It is something that should be checked in anyway.

Reply:
I think that's a good idea to put the version into somewhere else, such as a version file, so that a change of that version file will trigger the publish of the new version.

Reply:
What we do in the recommenders code is to [read it from a `__init__.py` file](https://github.com/microsoft/recommenders/blob/27709229cdc4aa7d39ab715789f093a2d21d2661/setup.py#L12) and whenever we want to publish a new release we edit this file.

Reply:
After considering there are Python and Scala packages, I move the version into the file [VERSION](https://github.com/simonzhaoms/recommenders/blob/simonz/sarplus/spark3/contrib/sarplus/VERSION), so that it can be used by both packages and specified in this single place.